### PR TITLE
[Xedra Evolved] Artifact looks_like things other than books

### PR DESCRIPTION
**Pull request by TheShadowFerret**
#### Summary
Mods "Xedra: Looks_like for artifacts"

#### Purpose of change

Most Xedra artifacts look like books, despite near-all not being in the shape of books.

#### Describe the solution

Add looks_like for artifacts.

#### Describe alternatives you've considered

Chunk of meat for the muscle artifact

#### Testing

Raid an office tower(which has a large amount of artifacts for some reason 🤔), look at artifacts.

#### Additional context

Occult seal says nothing about its physical description, so I didn't touch it.

### CODE REVIEW

File: data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_dreamsmith.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "id": "brass_cylinders",
     "type": "GENERIC",
     "category": "artifacts",
+    "looks_like": "gradcylinder"
     "name": { "str": "a set of brass cylinders", "str_pl": "sets of brass cylinders" },
```
Comment (Qrox) [inline data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_dreamsmith.json:7]:
```suggestion
    "name": { "str": "set of brass cylinders", "str_pl": "sets of brass cylinders" },
```

**Review by GuardianDll — COMMENTED** [review]
Thanks for your work

File: data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_classless.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
     "id": "magicians_hat",
     "type": "GENERIC",
     "category": "artifacts",
+    "looks_like": "tophat"
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_classless.json:6]:
```suggestion
    "looks_like": "tophat",
```

File: data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_dreamsmith.json [hunk 3]
```diff
@@ -3,7 +3,8 @@
     "id": "brass_cylinders",
     "type": "GENERIC",
     "category": "artifacts",
-    "name": { "str": "a set of brass cylinders", "str_pl": "sets of brass cylinders" },
+    "looks_like": "gradcylinder"
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_dreamsmith.json:6]:
```suggestion
    "looks_like": "gradcylinder",
```

File: data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_eater.json [hunk 4]
```diff
@@ -48,6 +50,7 @@
     "id": "glass_berries",
     "type": "GENERIC",
     "category": "artifacts",
+    "looks_like": "marble"
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_eater.json:53]:
```suggestion
    "looks_like": "marble",
```

**Review by Maleclypse — COMMENTED** [review]
Reviewdog appears to be down

**Pull request merged**